### PR TITLE
R4 - Update Encounter write fields

### PR DESCRIPTION
**Pull request by sbanda51**
Update documentation for R4 Encounter creates to remove locations, location.location fields and add serviceProvider as required field.

### CODE REVIEW

**Comment by MadhurRaj** [comment]
@sbanda51 any reason this is not following the PR template?

**Comment by MadhurRaj** [comment]
can you make the field_name_base_url a https:// link?

**Comment by MadhurRaj** [comment]
Screen Shot 2019-12-17 at 5 19 08 PM
Screen Shot 2019-12-17 at 5 24 02 PM

Local Screenshots before merge attached.

**Comment by sbanda51** [comment]
@MadhurRaj  what template are you referring to? It didn't gave me any auto template when opened the PR.

File: lib/resources/r4/encounter.yaml [hunk 1]
```diff
@@ -522,35 +522,18 @@ fields:
...
         info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-19-discharge-disposition
 
-- name: location
+- name: serviceProvider
   required: 'Yes'
-  cardinality: 0..*
-  type: BackboneElement
-  description: The locations where the patient has been.
+  cardinality: 0..1
+  type: Reference(Organization)
```
[the remark this answers is not in the document]
Comment (sbanda51) [reply lib/resources/r4/encounter.yaml]: done https://github.com/cerner/fhir.cerner.com/pull/335/commits/ce005c93ecf3a58b1693833d5f61e9dc9e757794

**Comment by MadhurRaj** [comment]
Oh...my mistake, I thought there was a PR template in place.

**Comment by averyallen** [comment]
image

**Comment by averyallen** [comment]
Evidence after merge:

Screen Shot 2019-12-19 at 7 58 09 AM
Screen Shot 2019-12-19 at 7 58 56 AM

**Pull request merged**